extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -0,0 +1,642 @@
+					if (!(this.handleOnTextChanged(this.vCoresMaxBox!, currentCPUSize!))) {
+						this.saveArgs!.coresRequest = undefined;
+					} else {
+						this.saveArgs!.coresRequest = this.vCoresMaxBox!.value;
+					}
+				})
+			);
+		} else if (type === loc.vCoresMin) {
+			currentCPUSize = this._postgresModel.config?.spec.scheduling?.default?.resources?.limits?.cpu;
+
+			if (!currentCPUSize) {
+				currentCPUSize = '0';
+			}
+
+			this.vCoresMinBox!.value = currentCPUSize;
+
+			this.disposables.push(
+				this.vCoresMinBox!.onTextChanged(() => {
+					if (!(this.handleOnTextChanged(this.vCoresMinBox!, currentCPUSize!))) {
+						this.saveArgs!.coresLimit = undefined;
+					} else {
+						this.saveArgs!.coresLimit = this.vCoresMinBox!.value;
+					}
+				})
+			);
+		}
+	}
+
+	private editMemory(type: string) {
+		let currentMemorySize: string | undefined;
+		let currentMemSizeConversion: string;
+
+		if (type === loc.memoryMax) {
+			currentMemorySize = this._postgresModel.config?.spec.scheduling?.default?.resources?.requests?.memory;
+
+			if (!currentMemorySize) {
+				currentMemSizeConversion = '0';
+			} else {
+				currentMemSizeConversion = this.gibibyteConversion(currentMemorySize);
+			}
+
+			this.memoryMaxBox!.value = currentMemSizeConversion!;
+
+			this.disposables.push(
+				this.memoryMaxBox!.onTextChanged(() => {
+					if (!(this.handleOnTextChanged(this.memoryMaxBox!, currentMemSizeConversion!))) {
+						this.saveArgs!.memoryRequest = undefined;
+					} else {
+						this.saveArgs!.memoryRequest = this.memoryMaxBox!.value + 'Gi';
+					}
+				})
+			);
+		} else if (type === loc.memoryMin) {
+			currentMemorySize = this._postgresModel.config?.spec.scheduling?.default?.resources?.limits?.memory;
+
+			if (!currentMemorySize) {
+				currentMemSizeConversion = '0';
+			} else {
+				currentMemSizeConversion = this.gibibyteConversion(currentMemorySize);
+			}
+
+			this.memoryMinBox!.value = currentMemSizeConversion!;
+
+			this.disposables.push(
+				this.memoryMinBox!.onTextChanged(() => {
+					if (!(this.handleOnTextChanged(this.memoryMinBox!, currentMemSizeConversion!))) {
+						this.saveArgs!.memoryLimit = undefined;
+					} else {
+						this.saveArgs!.memoryLimit = this.memoryMinBox!.value + 'Gi';
+					}
+				})
+			);
+		}
+	}
+
+	private gibibyteConversion(value: string): string {
+		let gtoGIConversion;
+		let floatValue = parseFloat(value);
+
+		let splitValue = value.split(String(floatValue));
+		let unit = splitValue[1];
+
+		if (unit === 'm') {
+			floatValue = (floatValue * Math.pow(10, 3)) / Math.pow(1024, 3);
+		} else if (unit === 'K') {
+			gtoGIConversion = 1000 / 1024;
+			floatValue = (floatValue * gtoGIConversion) / Math.pow(1024, 2);
+		} else if (unit === 'M') {
+			gtoGIConversion = Math.pow(1000, 2) / Math.pow(1024, 2);
+			floatValue = (floatValue * gtoGIConversion) / 1024;
+		} else if (unit === 'G') {
+			gtoGIConversion = Math.pow(1000, 3) / Math.pow(1024, 3);
```

Comment:
Nitpick: It's only a `gtoGIConversion` in this `G` branch.  The variable could be named something like `base10ToBase2Multiplier` so it makes sense for all units.